scripts/install-calico-windows.ps1

```diff
@@ -15,30 +12,16 @@ layout: null
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-{%- if site.prodname == "Calico" %}
-{%- assign installName = "Calico for Windows" %}
-{%- assign rootDir = "CalicoWindows" %}
-{%- assign zipFileName = "calico-windows.zip" %}
-{%- else %}
-{%- assign installName = "Tigera Calico for Windows" %}
-{%- assign rootDir = "TigeraCalico" %}
-{%- assign zipFileName = "tigera-calico-windows.zip" %}
-{%- endif %}
-
 <#
 .DESCRIPTION
-    This script installs and starts {{site.prodname}} services on a Windows node.
+    This script installs and starts Calico services on a Windows node.
```

Comment:
is there a reason to replace `{{site.prodname}` here? I would have thought we would have wanted this to carry over into our enterprise docs in the same way.

Reply:
The enterprise docs will have a generated version of `install-calico-windows.ps1` rendered and checked in. (It'll look very similar to the open source version.)